requirements.txt

```diff
@@ -21,7 +21,7 @@ python-json-logger==0.1.5
 redis==2.10.5
 requests==2.11.1
 setuptools==28.8.0
-Shapely==1.5.17
+Shapely==1.6.1
```

Comment:
Just curious: why do we need to upgrade Shapely?

Reply:
Shapely was giving an error with the travis build, Changing the version made the build pass